react/components/ContentWrapper.js

```diff
@@ -46,12 +46,22 @@ class ContentWrapper extends Component {
     return (
       <section className="vtex-account__page w-100 w-80-m">
         <header>
-          <PageHeader
-            title={title || intl.formatMessage({ id: titleId })}
-            {...(!hideBackButton && backButtonConfigs)}
-          >
-            {headerContent}
-          </PageHeader>
+          <div className="db dn-m">
+            <PageHeader
+              title={title || intl.formatMessage({ id: titleId })}
+              {...backButtonConfigs}
+            >
+              {headerContent}
+            </PageHeader>
+          </div>
+          <div className="db-m dn">
+            <PageHeader
+              title={title || intl.formatMessage({ id: titleId })}
+              {...(!hideBackButton && backButtonConfigs)}
+            >
+              {headerContent}
+            </PageHeader>
```

Comment:
These 2 `PageHeader`s are almost the same. Should both of them be here?

Reply:
Yes! the difference is that one as `db-m dn` as class an the other `db dn-m`, which makes the first one not visible in mobile and the second one only visible in mobile.